src/xcode/ENA/ENA/Source/Services/TicketValidation/TicketValidationError.swift

```diff
@@ -12,6 +12,7 @@ enum TicketValidationError: LocalizedError {
 	case accessToken(TicketValidationAccessTokenProcessingError)
 	case encryption(EncryptAndSignError)
 	case resultToken(TicketValidationResultTokenProcessingError)
+	case allowListError(AllowListError)
```

Comment:
Please make sure that the correct error message is used in the `errorDescription` func below